vault-core/src/main/java/org/apache/jackrabbit/vault/fs/api/FilterSet.java

```diff
@@ -239,12 +243,45 @@ public boolean covers(@NotNull String path) {
     }
 
     /**
-     * Checks if the given item is an ancestor of the root node.
+     * Checks if the given item is an ancestor of the filter's root path.
```

Comment:
s/given item/given path

Reply:
it's consistent with the method above; should I change both?

Reply:
I think what was meant was rather 
`Checks if the given path is an ancestor of the filter's root path`. A (string) path is given, not a JCR item!

Reply:
Yes. What I meant was: this is consistent with the Javadoc for other methods here. Should I use the opportunity to fix those as well?